liblangutil/Token.h

```diff
@@ -143,6 +143,7 @@ namespace solidity::langutil
 	/* Inline Assembly Operators */                                    \
 	T(AssemblyAssign, ":=", 2)                                         \
 	/* Keywords */                                                     \
+	K(Account, "account", 0)                                           \
```

Comment:
Little bit of bikeshedding here:
 - `owned`
 - `entrypoint`

Reply:
Agree that `account` is not ideal. I think I don't like `owned` though, seems like it is rather the opposite, no? Like AA contracts are in a sense self-owned, as opposed to EOAs?
Not sure about `entrypoint`, is that confusing with the entrypoint address?

Reply:
Or even changing it from a `contract` to an `account`?

Reply:
> Hm, how about `sovereign` or `autonomous` then?

I was also thinking about `sovereign`! I like it, but it is probably too opaque. Not sure about `autonomous`, that sounds like it is connected to DAOs.

Reply:
> Or even changing it from a `contract` to an `account`?

Just `account` instead of `account contract` is complicated. If I understand it correctly, `contract` right now is one of three possible types: `contract`, `interface`, `library`. It might be that you want to combine AA with several of these, e.g. `account interface`. That would be impossible if `account` instead is a fourth type...

Reply:
I think my preference would be to keep the keyword naming for the initial version, with the understanding that it is temporary. And then ask for suggestions on the discord after we publish our "local testnet MVP".